pkgs/applications/misc/kitty/default.nix

```diff
@@ -68,6 +68,7 @@ buildPythonApplication rec {
     })
   ] ++ stdenv.lib.optionals stdenv.isDarwin [
     ./macos-10.11.patch
+    ./macos-symlink.patch
```

Comment:
Can you use `fetchpatch` for this?

Reply:
https://github.com/kovidgoyal/kitty/commit/bdeec612667f6976109247fe1750b10dda9c24c0.patch